config/routes.rb

```diff
@@ -7,5 +7,14 @@
     end
     resources :sensors
     resources :sponsors
+
+    namespace :ios_app do
```

Comment:
I think this can be generalized to `app`, since I'll consume the new API from the Android app as well (and maybe even from the webapp).

Reply:
Let me call it `mobile_app` instead then.